.github/workflows/build.yaml

```diff
@@ -25,6 +30,7 @@ jobs:
       - name: test
         run: yarn test:ci
       - name: Coveralls
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
True. Here's the problem I know of. Because I also have relatively little experience with `CircleCI`. So a lazy way of writing is used here 😉

Reply:
But I can go and conduct an investigation at a later date and see if I can fix it perfectly

Reply:
I spent some time researching it.
In summary: **it can be done, but it's not necessary**

There is no code for `if (isWin){} / it.linux()` in our current unit tests.

So different platforms, they have the same result.  We just need the linux test results to complete the function.